[tgif] preserve non-forward method during torch package serialization

842974287 opened the pull request:
Reviewed By: terrycsy, sayitmemory

Differential Revision: D51607058

houseroad reviewed (approved):
Looks good.  Can we add a unittest?

842974287 committed:
[tgif] preserve non-forward method during torch package serialization (#114702)

Summary:

bypass-github-export-checks
force-merge-on-github

Reviewed By: terrycsy, houseroad, sayitmemory

Differential Revision: D51607058
